File: 5-Leccion-Arreglos-matrices/A.Tres-numeros-mas-grandes.java
import java.util.Scanner;
import java.util.Arrays;

public class Main {
  public static void main(String[] args) {
    Scanner scanner = new Scanner(System.in);

    int N = scanner.nextInt();
    int[] num = new int[N];

    for (int i = 0; i < N; i++)
      num[i] = scanner.nextInt();

    Arrays.sort(num);

    System.out.println(num[N - 1]);
    System.out.println(num[N - 2]);
    System.out.println(num[N - 3]);
  }
}

File: 5-Leccion-Arreglos-matrices/A.Tres-numeros-mas-grandes.cpp
#include <iostream>
#include <algorithm>
using namespace std;

int main() {
  int N;
  cin >> N;

  int num[N];

  for (int i = 0; i < N; i++)
    cin >> num[i];

  sort(num, num + N);

  cout << num[N - 1] << endl;
  cout << num[N - 2] << endl;
  cout << num[N - 3] << endl;

  return 0;
}